src/styles/blocks/contact.scss

```diff
@@ -0,0 +1,82 @@
+//CONTACT BLOCK:
+
+.contact {
+
+  &__value-name {
+    font-size: 14px;
+    line-height: 27px;
+    color: $tracperant-text;
+
+    margin-bottom: 8px;
+  }
+  &__value {
+    font-size: 16px;
+    line-height: 27px;
+    color: $regular-text-color;
+
+    margin-bottom: 24px;
+  }
+  &__input {
+    box-sizing: border-box;
```

Comment:
It's more practical to add `box-sizing: border-box;` to all elements on the page.

Reply:
> It's more practical to add `box-sizing: border-box;` to all elements on the page.

I fixed it but it doesnt marked as `Outdated `=(